### PR TITLE
Login & Signup restyled. See comment for details: Closes 57

### PR DESCRIPTION
**Pull request by swdonovan**
Login page and user signup page are restyled....Not a final style we will need to come up with a app wide backgrouund or whatever to replace the blue but the format is there and won't be hard to change. along with the css tags if we need to

### CODE REVIEW

**Comment by swdonovan** [comment]
All tests pass as well

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -1,3 +1,118 @@
...
+  background:url(https://i.ytimg.com/vi/4kfXjatgeEU/maxresdefault.jpg);
+}
+
+.main{
+ 	margin:50px 15px;
+}
+
+h1.title {
+	font-size: 50px;
+	font-family: 'Passion One', cursive;
```
Comment (MikelSage) [inline app/assets/stylesheets/users.scss:28]: I like the use of Passion One here.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -3,6 +3,15 @@
   <head>
     <title>ScootsShop</title>
     <%= csrf_meta_tags %>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+		<!-- Website CSS style -->
+		<link href="css/bootstrap.min.css" rel="stylesheet">
```
Comment (MikelSage) [inline app/views/layouts/application.html.erb:8]: Not quite sure about importing bootstrap again here, but if it works for now that's cool. We'll try playing around with it at a later date.

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -1,13 +1,40 @@
...
 
-<%= link_to "Create Account", new_user_path %>
+	</div>
+</div>
+</div>
+
+<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+<!-- Include all compiled plugins (below), or include individual files as needed -->
+<script src="js/bootstrap.min.js"></script>
```
Comment (MikelSage) [inline app/views/sessions/new.html.erb:40]: Same idea as with importing the bootstrap link tag, might try playing around to see if we can cut this out and still have everything work.

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -1,9 +1,48 @@
-<%= form_for @user do |f| %>
-  <%= f.label :username %>
-  <%= f.text_field :username %>
-  <%= f.label :password %>
-  <%= f.password_field :password %>
-  <%= f.label :email %>
-  <%= f.email_field :email %>
-  <%= f.submit "Get Ur Scoot On" %>
-<% end %>
+<div class="row main">
```
Comment (MikelSage) [inline app/views/users/new.html.erb:1]: I'm digging the way this form looks

**Review by MikelSage — APPROVED** [review]
image

**Pull request merged**